tests/helpers/test_storage.py

```diff
@@ -627,7 +627,7 @@ async def test_saving_load_round_trip(tmpdir: py.path.local) -> None:
     """Test saving and loading round trip."""
     loop = asyncio.get_running_loop()
     config_dir = await loop.run_in_executor(None, tmpdir.mkdir, "temp_storage")
-    async with async_test_home_assistant(config_dir=config_dir) as hass:
+    async with async_test_home_assistant(config_dir=str(config_dir)) as hass:
```

Comment:
LocalPath has a property for this:
```suggestion
    async with async_test_home_assistant(config_dir=config_dir.strpath) as hass:
```

Reply:
Adjusted in e5dfff41338ef7d314865fb20ff7c219a3d525a5